nvm.sh

```diff
@@ -2093,6 +2093,14 @@ nvm_get_download_slug() {
     fi
   fi
 
+  # If node version in below 16.0.0 then there is no arm64 packages available in node repositories, so we have to install "x64" arch packages
+  # If running MAC M1 :: arm64 arch and Darwin OS then use "x64" Architecture because node doesn't provide darwin_arm64 package below v16.0.0
+  if nvm_version_greater_than_or_equal_to '15.14.0' "${VERSION}"; then
```

Comment:
Ahhh so this branch is meant to be entered when it’s < 16? In that case:
```suggestion
  if nvm_version_greater '16.0.0' "${VERSION}"; then
```
should work?

Reply:
ok let me run some smoke test locally first

Reply:
It's the perfect suggestion. My bad, haven't looked at this function. Thank you 🙂